source/optional_features/digital_wallets/index.html.md.erb

```diff
@@ -0,0 +1,69 @@
+
+This is an invitation-only feature currently in private beta. [Contact the GOV.UK Pay team](https://docs.payments.service.gov.uk/support_contact_and_more_information/#contact-us) to request this feature for your service. After GOV.UK Pay has approved your request to participate in this private beta, GOV.UK Pay will email you a link to enable this feature. 
+
+You can only enable Google Pay and Apple Pay if you have a [live Worldpay account](/switching_to_live/set_up_a_live_worldpay_account/#set-up-a-live-worldpay-account).
+
+## Enable Apple Pay
+
+Sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/).
+
+1. Click on the link in the email from GOV.UK Pay 
+
+1. Select __Enable__ next to Apple Pay.
+
+1. On the __Enable Apple Pay__ screen, select __Yes, turn on Apple Pay__.
+
+## Enable Google Pay
+
+To enable Google Pay, you must enter a unique Google Pay merchant ID into your GOV.UK Pay account.
+
+You get your Google Pay merchant ID from your Worldpay merchant admin interface.
+
+1. If required, speak to your Worldpay account manager to upgrade your Worldpay merchant admin interface, so that the interface has Google Pay functionality.
+
+1. Sign in to your [Worldpay merchant admin interface](https://secure.worldpay.com/sso/public/auth/login.html) [external link].
+
+1. Go to __Integration__ and select the __Pay with Google__ tab.
+
+1. Under __Generate unique merchant ID__, select __Generate__.
+
+1. Copy the generated merchant ID.
+
+Sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/).
+
+1. Click on the link in the email from GOV.UK Pay 
+
+1. Select __Enable__ next to Google Pay.
+
+1. On the __Enable Google Pay__ screen, select __Yes, turn on Google Pay__.
+
+1. Enter the generated merchant ID and select __Submit__.
+
+## Secure authentication
+
+A user can securely authenticate digital wallet transactions that require secure authentication by either:
+
+* enabling secure authentication in their digital wallet
+* securely authenticating those transactions separately using [3D Secure authentication](https://www.pcisecuritystandards.org/documents/FAQs_for_PCI_3DS_Core_Security_Standard.pdf) [external link]
+
+Apple Pay enforces secure authentication, so users should not have to complete 3D Secure authentication during Apple Pay transactions.
+
+Google Pay does not enforce secure authentication. 
+
+## Restrictions
+
+The GOV.UK Pay platform cannot apply [corporate card surcharges](/optional_features/corporate_card_surcharges/#corporate_card_surcharges) to digital wallet transactions made with a corporate card.
+
+When you look at individual non-digital wallet transaction in your GOV.UK Pay account transactions list, you see the first six digits and last four digits of the card number. When you look at individual digital wallet transactions in your GOV.UK Pay account transactions list, you will only see the last 4 digits of the card number or device account number.
```

Comment:
Does “device account number” need defining? I guess we’re basically saying that they’ll see four digits but they might not necessarily match those on the physical card.

Also, inconsistent usage of “four” versus “4”.